extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/AutoScalingConfig.java

```diff
@@ -0,0 +1,49 @@
+
+package io.quarkus.kubernetes.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+
+@ConfigGroup
+public class AutoScalingConfig {
+
+    /**
+     * The Autoscaler class.
+     * Knative Serving comes with its own autoscaler, the KPA (Knative Pod Autoscaler) but can also be configured to use
+     * Kubernetes’ HPA (Horizontal Pod Autoscaler) or even a custom third-party autoscaler.
+     * Possible values (kpa, hpa, default: kpa).
```

Comment:
When you say default, you mean the default that knative itself uses if nothing is configured?

Reply:
Yes